lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyHoverParticipant.java

```diff
@@ -66,10 +69,32 @@ public Hover onText(IHoverRequest request, CancelChecker cancelChecker) {
             String featureName = request.getNode().getTextContent().trim();
             return getHoverFeatureDescription(featureName, request.getXMLDocument());
         }
+        if (LibertyConstants.PLATFORM_ELEMENT.equals(parentElement.getTagName())) {
+            String platformName = request.getNode().getTextContent().trim();
+            return getHoverPlatformDescription(platformName);
+        }
 
         return null;
     }
 
+    /**
+     * get description for platform from the feature json
+     * there will be a feature with same shortname as platform.
+     * @param platformName platform name
+     * @return hover
+     */
+    private Hover getHoverPlatformDescription(String platformName) {
+        String platformNameLowerCase = platformName.toLowerCase();
+        // strip off version number after the - so that we can provide all possible valid versions of a feature for completion
+        String featureNameToCompare = platformNameLowerCase.contains("-") ? platformNameLowerCase.substring(0, platformNameLowerCase.lastIndexOf("-")) : platformNameLowerCase;
```

Comment:
Fix the variable name and preceding comment to reference platform instead of feature.

Reply:
Done